src/interfaces/modular/SGBase.i

```diff
@@ -321,10 +329,13 @@ namespace std {
 #ifndef SWIGR
 %include <shogun/base/init.h>
 #endif
+%include <shogun/lib/basetag.h>
+%include <shogun/lib/tag.h>
 %include <shogun/base/SGObject.h>
 %include <shogun/io/SGIO.h>
 %include <shogun/base/Version.h>
 %include <shogun/base/Parallel.h>
+%include <../tests/unit/base/MockObject.h>
```

Comment:
this is still included without guards for testing...
and could you please use `#include "..."` instead of doing some magical relative path hacking for `#include <>`. note: read the manual of header include